forge/forge/op/eval/forge/transpose.py

```diff
@@ -63,67 +63,6 @@ def decompose(self, dc, inputs):
                 )
             )
 
-    def decompose_post_optimize(self, dc, inputs):
```

Comment:
Can we expand `forge/test/mlir/test_ops.py::test_transpose` op test to include -3 dims as well? 

It's okay to add lower compile depth for -3 (Z dim) cases until we uplift metal fix into FFE as well :))

Reply:
Added tests for -3 dims, as well as compile depth set to FINISH_COMPILE until metal uplift. Should I make an issue that reminds us to undo those changes?

Reply:
Yup! And feel free to assign it to yourself :))

Reply:
@mstojkovicTT did you try to run transpose on -3 without reducing compile depth? I think this will work too because we are doing some runtime decompositions... These are in: `third_party/tt-mlir/runtime/lib/ttnn/operations/data_movement/transpose.cpp`

Reply:
@dgolubovicTT that would work if i return from test before comparing with golden_pcc (before giving inputs to compiled model)

Reply:
update: i removed compile depth from the code, and just left return before giving inputs to the compiled_model